mdata/store_cassandra_test.go

```diff
@@ -0,0 +1,43 @@
+package mdata
+
+import (
+	"fmt"
+	"testing"
+)
+
+type testCase struct {
+	ttl                uint32
+	windowFactor       int
+	nameFormat         string
+	expectedTableName  string
+	expectedWindowSize uint32
+}
+
+const oneDay = 24 * 60 * 60
+const oneMonth = oneDay * 30
+const oneYear = oneMonth * 12
+
+func TestGetTTLTables(t *testing.T) {
+	tcs := []testCase{
+		{oneDay, 20, "metric_%d", "metric_16", 1},
+		{oneMonth, 20, "metric_%d", "metric_512", 26},
+		{oneYear, 20, "metric_%d", "metric_8192", 410},
+		{oneDay, 50, "metric_%d", "metric_16", 1},
+		{oneMonth, 50, "metric_%d", "metric_512", 11},
+		{oneYear, 50, "metric_%d", "metric_8192", 164},
```

Comment:
can you make this a bit more extensive and test some edge cases (like TTL of 0s, 10s, 1min, maxuint32, and on and around a boundary, for example 1024 hours minus a second, 1024 hours minus 2 seconds, 1024 hours, 1024 hours and a second)